go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/disintegration/imaging v1.6.2
 	github.com/dsnet/compress v0.0.1 // indirect
+	github.com/dsoprea/go-exif/v3 v3.0.0-20201216222538-db167117f483 // indirect
```

Comment:
Why not stick to version v3.0.0?

Reply:
Honestly no clue. That was added by `go get` automatically...

Reply:
Fix it, please

Reply:
I looked into this and tried to stick to the version number you mentioned. However, the library does not seem provide tagged releases:

```
:~/Code/filebrowser*master$ go get -d github.com/dsoprea/go-exif/v3@v3.0.0
go get: github.com/dsoprea/go-exif/v3@v3.0.0: invalid version: unknown revision v3.0.0

:~/Code/filebrowser*master$ go get -d github.com/dsoprea/go-exif/v3@3.0.0
go: downloading github.com/dsoprea/go-exif v0.0.0-20200711180620-208788de2815
go get: module github.com/dsoprea/go-exif@3.0.0 found (v0.0.0-20200711180620-208788de2815), but does not contain package github.com/dsoprea/go-exif/v3
```

The library is pulled as recommended. Other projects use the library in a similar way. Examples are PhotoPrism (listed on [Awesome-Selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted)) and other projects on GitHub:

* [PhotoPrism](https://github.com/photoprism/photoprism/blob/develop/go.mod)
* [GoGallery](https://github.com/robrotheram/gogallery/blob/master/server/go.mod)
* [Collage](https://github.com/fawick/collage/blob/master/go.mod)
* [photoimportd](https://github.com/bwdezend/photoimportd/blob/master/go.mod)

As such, the dependency file should probably be kept as committed.